library/core/src/macros/mod.rs

```diff
@@ -713,7 +713,7 @@ macro_rules! unimplemented {
 /// Indicates unfinished code.
 ///
 /// This can be useful if you are prototyping and are just looking to have your
-/// code typecheck.
+/// code typechecked.
```

Comment:
Maybe we should avoid that jargon altogether? Something like:

> This can be useful if you are prototyping and just want a placeholder to let your code pass type analysis.

Reply:
There's no reason for me not to use your words verbatim. Updating the PR.